src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/resource/ClientsController.kt

```diff
@@ -134,6 +134,12 @@ class ClientsController(
     )
     telemetryClient.trackEvent("AuthorizationServerClientDeploymentDetailsUpsert", telemetryMap)
   }
+
```

Comment:
I think you added a `MigrationController` under your last ticket - given that this end point should only be required for the migration period, probably better to put it in there? Thinking a `MigrationService` might also be a good idea i.e. to keep all of the temporary code concerned only with migration isolated?

Reply:
Thanks, Updated